### PR TITLE
CY-3094 REST-client calls for forcing plugin installation

### PR DESCRIPTION
**Pull request by tehasdf**

### CODE REVIEW

File: cloudify/dispatch.py [hunk 1]
```diff
@@ -98,7 +98,9 @@
     'script',
     'cfy_extensions',
     'default_workflows',
-    'worker_installer'
+    'worker_installer',
+    'cloudify_system_workflows'
+
```
Comment (geokala) [inline cloudify/dispatch.py:103]: Well, I do like trailing something- but generally comma, not extra newline.

File: cloudify/context.py [hunk 2]
```diff
@@ -352,6 +352,11 @@ def id(self):
         """The node instance id."""
         return self._context.get('node_id')
 
+    @property
```
Comment (geokala) [inline cloudify/context.py:355]: Any reason not to just make self._node be self.node?

File: cloudify/models_states.py [hunk 3]
```diff
@@ -93,3 +93,13 @@ class AgentState(object):
     STATES = [CREATING, CREATED, CONFIGURING, CONFIGURED, STARTING, STARTED,
               STOPPING, STOPPED, DELETING, DELETED, RESTARTING, RESTARTED,
               UPGRADING, UPGRADED, FAILED, NONRESPONSIVE, RESTORED]
+
+
+class PluginInstallationState(object):
+    PENDING = 'pending'
```
Comment (geokala) [inline cloudify/models_states.py:99]: Would it makes sense to have this be pending-install, to mirror pending-uninstall?

File: cloudify/context.py (hunk 2 above)
Comment (tehasdf) [reply cloudify/context.py:355]: consistency with all the other ones

Comment (tehasdf) [reply cloudify/context.py:355]: and i guess self._node is a restclient node object, only used internally in this, and this is a NodeContext

**Pull request merged**